### PR TITLE
Add Smart Converter v2.2.3

### LINKED ISSUE #21660: Cask request: Smart Converter
### Cask details

**Name** Smart Converter

**Homepage** http://shedworx.com/smart-converter-pro

**License** -

**Download URL** http://download.shedworx.com/smartconverter/SmartConverter-2.2.3.dmg

**Description**
Smart Converter is the fastest, easiest to use video converter on the App Store, thanks to it's Smart Conversion technology.

Personally i use it in the free version to convert a few videos to format like DivX

### PR DESCRIPTION
**Pull request by derimagia**
#### Adding a new cask
- [X] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] Commit message includes cask’s name.
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` left no offenses.
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.

Fixes #21660 

General question - I don't see this app having https, do we have any warnings or any standards for this? Don't think it's possible to convert everything to https but would like to know

### CODE REVIEW

**Comment by derimagia** [comment]
Added Zap Stanza

File: Casks/smart-converter.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'smart-converter' do
```
Comment (vitorgalvao) [inline Casks/smart-converter.rb]: Token reference was not followed. This needs to be `smartconverter`.

**Comment by vitorgalvao** [comment]
> I don't see this app having https, do we have any warnings or any standards for this?

No. If they don’t have it, they don’t have it.

**Comment by derimagia** [comment]
Shame there isn't a warning.  And yeah I had it that way but then changed
it to match the name.  There a reason this isn't in the audit script?

**Comment by derimagia** [comment]
Updated

**Comment by vitorgalvao** [comment]
> There a reason this isn't in the audit script?

Yes, there’s too many edge cases. Read the linked documentation, and you’ll see them.

**Pull request merged**